olive/packaging/sample_code/ONNXModel/cpp/code_sample.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+-------------------------------------------------------------------------
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+--------------------------------------------------------------------------
+*/
+#include <iostream>
+#include <fstream>
+#include "nlohmann/json.hpp"
+#include "onnxruntime_cxx_api.h"
+
+
+void updateSessOptions(SessionOptions& sess_options, const nlohmann::json& session_options);
+void updateExecutionProvider(SessionOptions& sess_options, const nlohmann::json& executionProvider_json);
+
+
+int main() {
+    std::ifstream i("inference_config.json");
```

Comment:
The sample code cannot be simply copied and pasted to use because real code needs data input to inference. As for CPP and CS part sample codes, I consulted with Baiju and Dmitri, and they confirmed the sample codes are good from the ONNXRuntime API point of view.

Reply:
What if ONNXRuntime API changes? We should have unit tests in our repo to check ONNXRuntime API used in sample code.

Reply:
ok